### PR TITLE
enable more CentOS dualstack e2e tests

### LINKED ISSUE #9994: Investigate unsupported dual-stack OS/provider combinations
Investigate the unsupported combinations of OS/provider for dual-stack, [commented-out in the e2e test](https://github.com/kubermatic/kubermatic/blob/3631688b512da07ca973f3acf43be292da8e22d2/pkg/test/dualstack/cloud_providers_dualstack_test.go#L94)

### PR DESCRIPTION
**Pull request by PratikDeoghare**
**What does this PR do / Why do we need it**:

**Does this PR close any issues?**:
Fixes #9994

**Special notes for your reviewer**:

**Documentation**:

**Does this PR introduce a user-facing change?**:

```release-note
```

### CODE REVIEW

File: pkg/test/dualstack/cloud_providers_dualstack_test.go [hunk 1]
```diff
@@ -131,11 +131,13 @@ func TestCloudClusterIPFamily(t *testing.T) {
...
 			},
-			cni:                 "canal",
-			ipFamily:            util.DualStack,
-			skipNodes:           true,
-			skipHostNetworkPods: true,
+			cni:                    "canal",
+			ipFamily:               util.DualStack,
+			skipNodes:              true,
+			skipHostNetworkPods:    true,
+			skipEgressConnectivity: true,
```
Comment (rastislavs) [inline pkg/test/dualstack/cloud_providers_dualstack_test.go:140]: Why egress connectivity is not working on AWS, if it was before?

Comment (PratikDeoghare) [reply pkg/test/dualstack/cloud_providers_dualstack_test.go:140]: Egress doesn't work for CentOS on AWS. 

But this change disables them for both. That is not good! :grimacing: 

Lemme check whats up there.

**Pull request closed without merging**